src/main/java/com/minecolonies/coremod/client/gui/WindowTownHallColonyManage.java

```diff
@@ -0,0 +1,192 @@
+            }
+            else
+            {
+                // No close colony
+                findPaneOfTypeByID(TEXT_NEARBY, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.nonenearby"));
+            }
+        }
+
+        final IColony ownerColony = IColonyManager.getInstance().getIColonyByOwner(world, player);
+        if (ownerColony != null)
+        {
+            findPaneOfTypeByID(BUTTON_DELETE, ButtonImage.class).enable();
+            findPaneOfTypeByID(TEXT_OWN, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.own", ownerColony.getCenter()));
+
+            if (MineColonies.getConfig().getCommon().allowInfiniteColonies.get())
+            {
+                findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.denied.existingandabandon"));
+            }
+            else
+            {
+                findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.denied.existing"));
+            }
+        }
+        else
+        {
+            findPaneOfTypeByID(TEXT_OWN, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.none"));
+
+            if (existingColony != null || IColonyManager.getInstance().isTooCloseToColony(world, pos))
+            {
+                findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.denied.tooclose"));
+            }
+        }
+
+        if (MineColonies.getConfig().getCommon().restrictColonyPlacement.get())
+        {
+            final double spawnDistance = Math.sqrt(BlockPosUtil.getDistanceSquared2D(pos, world.getSpawnPoint()));
+            if (spawnDistance < MineColonies.getConfig().getCommon().minDistanceFromWorldSpawn.get())
+            {
+                findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format(CANT_PLACE_COLONY_TOO_CLOSE_TO_SPAWN,
+                  MineColonies.getConfig().getCommon().minDistanceFromWorldSpawn.get()));
+            }
+            else if (spawnDistance > MineColonies.getConfig().getCommon().maxDistanceFromWorldSpawn.get())
+            {
+                findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format(CANT_PLACE_COLONY_TOO_FAR_FROM_SPAWN,
+                  MineColonies.getConfig().getCommon().maxDistanceFromWorldSpawn.get()));
+            }
+        }
+
+        if (findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).getTextContent().isEmpty())
+        {
+            findPaneOfTypeByID(BUTTON_CREATE, ButtonImage.class).enable();
+            findPaneOfTypeByID(TEXT_FEEDBACK, Text.class).setTextContent(LanguageHandler.format("com.minecolonies.coremod.gui.colony.allowed.create"));
+        }
+    }
+
+    @Override
+    public void onButtonClicked(@NotNull final Button button)
```

Comment:
Why not register them as button handlers?

Reply:
fixed